interface/main/tabs/main.php

```diff
@@ -289,4 +289,31 @@ function isEncounterLocked( encounterId ) {
 $('#updater-icon').click(function () {
     $('#updater-iframe').iziModal('open');
 });
+
+function showUpdaterNotifications(type, title, message) {
```

Comment:
recommend functions stored in feature's directory, and conditional as per globals config.

Reply:
Just for testing purpose i placed it here, once i finish working on the updater_settings table i will trigger this functions upon the conditionals